README.md

```diff
@@ -1,2 +1,123 @@
-# sensu-handler-pagerduty-alternative
-An alternative handler for communicating with pager duty from Sensu (sensu-go)
+# Alternative Sensu PagerDuty Handler
+
+
+## Table of Contents
+
+ - [Overview](#overview)
+ - [Usage examples](#usage-examples)
+   - [Help output](#help-output)
+   - [Deduplication key](#deduplication-key)
+   - [Summary](#summary)
+   - [Source](#source)
+   - [Status](#status)
+   - [Details](#details)
+ - [Configuration](#configuration)
+   - [Installation](#installation)
+   - [Handler definition](#handler-definition)
+ - [Environment variables](#environment-variables)
+
+## Overview
+
+This Python package provides an alternative [Sensu Go](https://sensu.io/) [handler](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-process/handlers/) for communicating with [PagerDuty](https://pagerduty.com/).  Originally authored under the auspices of my day job at [DigitalOcean](https://www.digitalocean.com/) and released as open source with their gracious permission.
+
+While there is a [handler](https://github.com/sensu/sensu-pagerduty-handler) released by the good folks at [Sensu](https://sensu.io/), there was one piece of functionality it did not have that I needed.  Specifically, I had the need to be able to customize the data structure that is sent to [PagerDuty](https://pagerduty.com/) in the customer details section of an incident.
```

Comment:
Is there a reason that this shouldn't be added as functionality to a fork of that and a PR opened to that project?